tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -114,7 +111,7 @@ public function testMessages()
             ->utf8('column')
             ->utf8Extended('column')
             ->integer('column')
-            ->isArray('column')
+            ->array('column')
```

Comment:
`Validator::isArray()` is deprecated in CakePHP5. Need to use `Validator::array()` instead.
Refer: https://book.cakephp.org/4/en/appendices/4-5-migration-guide.html#validation